src/errors.ts

```diff
@@ -0,0 +1,11 @@
+/**
+ * Convenience function for constructing errors where the `name` property has a
+ * given value. This enables easier identification and grouping in e.g. Sentry.
+ */
+export const exception = (name: string, message: string) => {
+  const error = new Error(message);
+  error.name = name;
```

Comment:
suggestion: instead of using a helper function, would it be possible to change the `name` property in the `Error` subclasses you created before?

Reply:
As far as I can tell, there's no straightforward way to do this other than typing out something like:

```js
class MyError extends Error {
  function constructor(...args) {
    super(...args);
    this.name = "MyError";
    Error.captureStackTrace(this, this.constructor);
  }
}
```

...for every error class you want to have. Given I'm not sure this approach has any advantages over the current one (do you know of any?), I'd prefer to avoid the noise and duplication.

Reply:
Hmmm, I was hoping it could be done somehow without having to write a constructor. This feels like a weird pattern to me coming from other object-oriented languages with exceptions, but if it's the simplest way to achieve this with Sentry then it makes sense. Is the specific Sentry issue documented anywhere? I tried looking at `ocs_saver` in our Sentry account but didn't see any errors.

Reply:
I don't know if it's explicitly documented that Sentry uses the `name` property of errors to title and group them. I might not call it an "issue" exactly — from what I've seen, other libraries and even Node itself use this property to identify errors.